admin/spec/controllers/spree/admin/stores_controller_spec.rb

```
@@ -52,7 +52,12 @@
         name: 'New Store Name',
         default_currency: 'GBR',
         default_country_iso: 'GB',
-        default_locale: 'pl'
+ expect(store.default_currency).to eq('GBR')
```

RuboCop (1.75.5)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 3.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)